pom.xml

```diff
@@ -805,7 +811,8 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-shade-plugin</artifactId>
-          <version>2.2</version>
+          <!-- Needs to be 2.4.2 or later to prevent MSHADE-148 -->
+          <version>2.4.3</version>
```

Comment:
I don't really find any side effect so far. 
Without this upgrade, the build fails non-deterministically.